.github/workflows/lint.yml

```diff
@@ -2,10 +2,9 @@ name: ESLint Check
 
 on:
   push:
-    branches:
-      - ["main", "staging"]
-    pull_request:
-      - ["main", "staging"]
+    branches: ["**"]
+  pull_request:
+    branches: ["**"]
 
 jobs:
   lint:
```

Comment:
Hello bro. You did a nice job here. What do you think if we can add more linting steps this. Something like
`steps:
    - name: Checkout code
      uses: actions/checkout@v2

    - name: Install dependencies
      run: npm install

    - name: Run linting
      run: npm run lint`